lbz/misc.py

```diff
@@ -114,7 +115,7 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
-def copy_without_keys(data: dict, *keys) -> dict:
+def copy_without_keys(data: Union[dict, MultiDict], *keys) -> dict:
```

Comment:
I think it could be done better, by accepting all dict subclases, not sure tho

Reply:
that depends on usage out side LBZ.

Reply:
It was pointed in the previous [pull reqeust](https://github.com/pdyba/lambdalizator/pull/35#discussion_r672866321) - still, I think changing the type of input argument is not that good idea 🤔

Reply:
in lbz/resource.py:211 we are using it and passing MutliDict

Reply:
Ok, interesting, so, according to the original comment, it should be more generic: `data: MutableMapping` 😉 

And, it would be nice to point it more directly in the docstring that the return value is a simple dict, nothing else.

Reply:
Also, the code in method you pointed looks buggy 🤔 
```python
>>> urlencode({"city": ["Berlin", "Paris"]})
'city=%5B%27Berlin%27%2C+%27Paris%27%5D'
>>> urlencode({"city": "Berlin", "city": "Paris"})
'city=Paris'
```
Correct me if I am wrong but I would say, we need to fix that sooner than later.

Reply:
Fixed in another PR.